pkg/crypto/keys/publickey.go

```diff
@@ -230,10 +231,13 @@ func (p *PublicKey) EncodeBinary(w *io.BinWriter) {
 // public key.
 func (p *PublicKey) GetVerificationScript() []byte {
 	b := p.Bytes()
-	b = append([]byte{byte(opcode.PUSHBYTES33)}, b...)
-	b = append(b, byte(opcode.CHECKSIG))
+	buf := io.NewBufBinWriter()
+	emit.Instruction(buf.BinWriter, opcode.PUSHDATA1, []byte{33})
+	buf.BinWriter.WriteBytes(b)
```

Comment:
Maybe we can use `emit.Bytes()` here? It will use `PUSHBYTES33` while we still have it and then can be adapted to newer PUSH instructions set.

Reply:
Changing verification script requires converting and changing all WIFs across the codebase which is rather annoying.
We can replace it with `emit.Bytes` later without touching script hashes or addresses.

I have first changed `Bytes()` to not emit `PUSHBYTES*` but there were many changes in other parts of code, so I decided to leave it for later.

Reply:
OK, let it be this way for a while.